system/Helpers/text_helper.php

```diff
@@ -756,9 +756,8 @@ function random_string(string $type = 'alnum', int $len = 8): string
 
 				return substr(str_shuffle(str_repeat($pool, ceil($len / strlen($pool)))), 0, $len);
 			case 'md5':
-				return md5(uniqid(mt_rand(), true));
 			case 'sha1':
-				return sha1(uniqid(mt_rand(), true));
+				return $type(uniqid(mt_rand(), true));
```

Comment:
This may be more "efficient", but the original is clearer

Reply:
Reverted the changes